test/test_bin.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import pytest
+from hisp.bin import FWBin3Subs, FWBin2Subs, DivBin, BinCollection, Reactor
+
+# create Reactor
+fw_bins = [FWBin3Subs() for _ in range(18)]
+fw_bins[1] = FWBin2Subs()
+div_bins = [DivBin() for _ in range(18, 46)]
+
+FW_bins = BinCollection(fw_bins)
+Div_bins = BinCollection(div_bins)
+
+my_reactor = Reactor(first_wall=FW_bins, divertor=Div_bins)
+
+# make bins
+for bin_index in list(range(18)):
+    fw_bin = FW_bins.get_bin(bin_index)
+    for subbin in fw_bin.sub_bins:
+        subbin.thickness = 6e-3
+        subbin.material = "W"
+
+for bin_index in list(range(18,46)):
+    div_bin = Div_bins.get_bin(bin_index)
+    div_bin.thickness = 1e-6
+    div_bin.material = "B"
+    div_bin.set_inner_and_outer_bins()
+
+filename = "test/wetted_test_data.csv"
+my_reactor.read_wetted_data(filename)
+
+@pytest.mark.parametrize("Slow, Stot, Shigh,f", [(2,3,1,0.5), (1,10, None, None)])
+def test_wetted_fraction(Slow,Stot,Shigh,f):
```

Comment:
I don't understand why Slow Stot etc are parameters here? should they not be read from the CSV file???